Packs/XForceExchange/Integrations/XFE_v2/XFE_v2.py

```diff
@@ -34,8 +34,13 @@ def ip_report(self, ip: str) -> dict:
 
         return self._http_request('GET', f'/ipr/{ip}')
 
-    def url_report(self, url: str) -> dict:
-        return self._http_request('GET', f'/url/{url}').get('result')
+    def url_report(self, url: str):
+        try:
+            response = self._http_request('GET', f'/url/{url}')
+        except Exception as e:
+            if "Not Found" in str(e):
+                return "Not Found"

```

Comment:
raise the exception again if the condition is not satisfied.
```suggestion
                return "Not Found"
            raise
```